kwords.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+
+url = "http://0.0.0.0:5000/"
+
+def get_keywords(text):
+
+    payload = json.dumps({
+    "text": text,
+    "lang": "en",
+    "n": 10
+    })
+    headers = {
+    'Content-Type': 'application/json'
+    }
+
+    response = requests.request("POST", url, headers=headers, data=payload)
```

Comment:
**Suggestion:** The 'requests.request' method could throw exceptions, such as a 'requests.exceptions.RequestException'. It would be a good practice to handle these potential exceptions to avoid unexpected crashes.
```suggestion
    try:
        response = requests.request("POST", url, headers=headers, data=payload)
    except requests.exceptions.RequestException as e:
        print(f'An error occurred: {e}')
        return None
```